ebuku.el

```diff
@@ -853,6 +853,49 @@ If an argument is excluded, get it from `ebuku-cache-default-args'."
       (setq tags (nconc (map-elt bookmark 'tags) tags)))
     (setq ebuku-tags (sort (seq-uniq tags) 'string-collate-lessp))))
 
+(defun ebuku-append-tag-to-bookmark ()
+  "Interactive function to append tag to bookmark."
+  ;; TODO fix an issue where it moves the cursor to the top after adding
+  (interactive)
+  (ebuku-update-tags-cache)
+  (let ((tag (completing-read "Append tag? " ebuku-tags))
+	(index (ebuku--get-index-at-point))
+	(inhibit-read-only t))
+    (ebuku--tags-add index tag)
+    (ebuku-refresh)
+    (message "Added tag to bookmark")))
+
+(defun ebuku--tags-join (tags)
+  "Internal function that joins a list of tags into a comma separated string of tags."
+  (typecase tags
+    (stringp tags)
+    (listp (mapconcat 'identity tags ","))))
+
+(defun ebuku--tags-add (index tags)
+  "Append tag to the list of tags on the index"
+  (with-temp-buffer
+    (ebuku--call-buku
+     `("--update" , index
+       "--tag", "+" , (ebuku--tags-join tags)))))
+
+(defun ebuku--tags-remove (index tags)
+  "Remove tag from the list of tags on the index"
+  (with-temp-buffer
+    (ebuku--call-buku
+     `("--update" , index
+       "--tag", "-" , (ebuku--tags-join tags)))))
+
+(defun ebuku--tags-set (index tags)
+  "Replace the current list of tags on the index with the one provided to this bookmark."
+  (with-temp-buffer
+    (ebuku--call-buku-no-output
```

Comment:
There's no function `ebuku--call-buku-no-output` in your PR. :-)

Reply:
Good catch. Forgot to change it back to `ebuku--call-buku`, I was playing around with a second `ebuku--call-buku` function but ultimately decided to remove it before pushing as I like making my first pull request to any project as small in scope as possible. Especially since this is my first contribution for elisp.